index.d.ts

```diff
@@ -33,21 +33,23 @@ export declare class StateObservable<S> extends Observable<S> {
   value: S
 }
 
-export declare interface Epic<Input extends Action = any, Output extends Input = Input, State = any, Dependencies = any> {
+type State = any;
```

Comment:
This name is being shadowed now. Do we need to declare this separately? I think I'd prefer to use `any` directly.

Reply:
I thought it was better to be explicit that the parameter is for State, hence the type. I can revert this if required

Reply:
If you're okay with it, I think we should use `any` explicitly rather than an alias of `type State = any`. Cool?